### PR TITLE
{bio}[GCC/12.3.0] SingleM v0.16.0, smafa v0.8.0, psycopg v3.2.1, ...

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/235
resolves https://github.com/vscentrum/vsc-software-stack/issues/367 (CD-HIT)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+files_to_copy = [(local_list_of_executables, 'bin'), (['*.pl'], 'bin'), 'README', 'doc', 'license.txt']
+
+fix_perl_shebang_for = ['bin/*.pl']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in local_list_of_executables],
+    'dirs': [],
+}
+
```
Comment (laraPPr) [inline easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-GCC-12.3.0.eb:37]: Could you add a `sanity_check_command` was there in other easyconfigs

Comment (laraPPr) [inline easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-GCC-12.3.0.eb:37]: Could you add a `sanity_check_command` was there in other easyconfigs

**Review by laraPPr — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-GCC-12.3.0.eb (hunk 1 above)
Comment (pavelToman) [reply easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-GCC-12.3.0.eb:38]: sanity_check_commands added

**Comment by laraPPr** [comment]
Test report by @laraPPr
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in total)
node3510.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.11.3
See https://gist.github.com/laraPPr/2ee62c6d528e56c916e0fbe9360aabf5 for a full test report.

**Comment by laraPPr** [comment]
missing dependency will run again in a minute

**Comment by laraPPr** [comment]
having some troubles with polars so will come back to it tomorrow

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3509.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.11.3
See https://gist.github.com/laraPPr/1770ec019bc64ff481f54a0f57d7a6a3 for a full test report.

**Review by laraPPr — APPROVED** [review]
lgtm

**Comment by laraPPr** [comment]
@boegelbot please test @ generoso

**Comment by laraPPr** [comment]
@boegelbot please test @ jsc-zen3

**Comment by laraPPr** [comment]
Going in, thanks @pavelToman!

**Pull request merged**